mintapi/cli.py

```diff
@@ -411,42 +419,47 @@ def main():
     data = None
     if options.accounts and options.budgets:
         try:
-            data = mint.get_account_data()
+            data = mint.get_account_data(limit=options.limit)
         except Exception:
             accounts = None
 
         try:
-            budgets = mint.get_budgets()
+            budgets = mint.get_budgets(limit=options.limit)
         except Exception:
             budgets = None
 
         data = {"accounts": accounts, "budgets": budgets}
     elif options.budgets:
         try:
-            data = mint.get_budgets()
+            data = mint.get_budgets(limit=options.limit)
         except Exception:
             data = None
     elif options.budget_hist:
         try:
-            data = mint.get_budgets(hist=12)
+            data = mint.get_budgets(limit=options.limit, hist=12)
         except Exception:
             data = None
     elif options.accounts:
         try:
-            data = mint.get_account_data()
+            data = mint.get_account_data(options.limit)
```

Comment:
Should we specify this by keyword, `limit=options.limit`, for consistency? Otherwise looks great!

Reply:
@mrooney yup, I just missed that one. Thanks! I'll get that fixed up tonight.